credstash.py

```diff
@@ -277,7 +277,7 @@ def putSecret(name, secret, version="", kms_key="alias/credstash",
     sealed = seal_aes_ctr_legacy(
         key_service,
         secret,
-        **kwargs
+        digest_method=digest,
```

Comment:
`seal_aes_ctr_legacy` accepts only 3 arguments
* key_service
* secret
* digest_method

all of them are present in call.